tests/TestCase/Validation/ValidatorTest.php

```diff
@@ -164,6 +189,33 @@ public function testErrorsWithPresenceRequired()
     }
 
     /**
+     * Test that errors() can work with nested data.
+     *
+     * @return void
+     */
+    public function testErrorsWithNestedFields()
+    {
+        $validator = new Validator;
+        $validator->add('user.username', 'letter', ['rule' => 'alphanumeric']);
+        $validator->add('comments.0.comment', 'letter', ['rule' => 'alphanumeric']);
```

Comment:
How would that work for the N row case? Would we have separate methods for single objects being nested, and multiples?